_posts/2019-11-21-we-are-striking-really.md

```diff
@@ -0,0 +1,22 @@
+---
+site: freiburg
+title: All has been said - act!
```

Comment:
```suggestion
title: Enough talking - time to act!
```

Reply:
Well, the current title is the slogan of a recent scientists protest in Berlin (https://www.scientists4future.org/2019/10/es-wurde-alles-gesagt-handeln/) - that's why I chose it. But yours isn't bad either.

Reply:
Ah okay that makes sense. It just sounds a little strange translated directly into English in my view.